packages/components/src/toggle-group-control/test/index.js

```diff
@@ -34,7 +34,7 @@ describe( 'ToggleGroupControl', () => {
 				{ options }
 			</ToggleGroupControl>
 		);
-		const firstButton = getByLabelText( 'R' );
+		const firstButton = getAllByText( 'R' )[ 0 ];
```

Comment:
Let's also check the accessible role of the button:

```suggestion
		const firstButton = screen.getByRole( 'radio', { name: 'R' } );
```

Reply:
Good one!